### PR TITLE
doc: document web flow and add details to otp flow

### PR DESCRIPTION
**Pull request by eoln**
more details needed by DFSP Mock Login Page/Simulator

### CODE REVIEW

File: docs/linking/README.md [hunk 1]
```diff
@@ -162,6 +176,10 @@ should prompt the user for this secret and then provide that back to the DFSP.
 
 ![Authentication (OTP)](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/mojaloop/pisp/master/docs/linking/3-authentication-otp.puml)
 
+#### DFSPAuthorizeSimulator
+FSPAuthorizeSimulator is api service for demo purposes. For the `OTP flow` it exposes `/sendOTP` API endpoint which allows to simulate sending SMS token to the User's device.
```
Comment (kleyow) [inline docs/linking/README.md:180]: nit: DFSP instead of FSP

**Review by kleyow — DISMISSED** [review]

File: docs/linking/2-request-consent-web.puml [hunk 2]
```diff
@@ -10,6 +10,11 @@ end box
 
 participant DFSP
 
+participant DFSPAuthorizeSimulator
+participant WebBrowser
+
+actor User
```
Comment (lewisdaly) [inline docs/linking/2-request-consent-web.puml:16]: Can `WebBrowser` and `User`  please be on the far left? That will line up better with our other sequences

**Review by lewisdaly — CHANGES_REQUESTED** [review]
This looks great, especially the clarity it adds to the web flow. 

Just a minor nitpick on the participant ordering, but nothing major.

File: docs/linking/2-request-consent-web.puml (hunk 2 above)
Comment (eoln) [reply docs/linking/2-request-consent-web.puml:13]: done

Comment (lewisdaly) [reply docs/linking/2-request-consent-web.puml:13]: Hmm I don't think the svg rerendered

File: docs/linking/README.md [hunk 3]
```diff
@@ -162,6 +176,10 @@ should prompt the user for this secret and then provide that back to the DFSP.
 
 ![Authentication (OTP)](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/mojaloop/pisp/master/docs/linking/3-authentication-otp.puml)
 
+#### DFSPAuthorizeSimulator
+DFSPAuthorizeSimulator is api service for demo purposes. For the `OTP flow` it exposes `/sendOTP` API endpoint which allows to simulate sending SMS token to the User's device.
+
+It is implemented as extension for TestingToolkit service which allows to define testing scenarios as TTK rules driven by `username` parameter.
```
Comment (lewisdaly) [inline docs/linking/README.md:182]: I'm not quite sure I see how this works. Will there be a way for the PISP to configure the username? Or will it just be the DFSP?

Comment (eoln) [reply docs/linking/README.md:182]: @lewisdaly `sendOTP` is API used by DFSP, we added it so we can simulate sending SMS in the OTP flow, we plan to first implement it in TTK, later we can use AWS SNS to send real SMS - or any other service if needed

File: docs/linking/2-request-consent-web.puml (hunk 2 above)
Comment (eoln) [reply docs/linking/2-request-consent-web.puml:13]: and now? I see the latest svg file

File: docs/linking/README.md (hunk 3 above)
Comment (eoln) [reply docs/linking/README.md:183]: @lewisdaly with @sridharvoruganti we decided that we will implement that DFSP needs to store consentRequest details somewhere - actions SIM-1..3 - here in DFSPAuthorizeSimulator, so later the DFSPLoginApp can use `GET /consentRequests/<id>` to retrieve the details, including the list of accounts related to this consent request grant SIM-12..15

Comment (eoln) [reply docs/linking/README.md:183]: story to implement generic store in TTK: https://app.zenhub.com/workspaces/mojaloop-project-59edee71d1407922110cf083/issues/mojaloop/project/1986

**Review by lewisdaly — APPROVED** [review]

**Pull request merged**